Video testimonial section

YilKanda opened the pull request:
# Description
I made a section where you can see a testimony video of one of the first graduates of the BrightCoders program. 

With this Pull Request it is expected to solve the need raised in issue [#20](https://github.com/bright-coders/bc-landing-page-2021/issues/20).
 
# Running
## Install the required gems

Run the command:

`bundle `

or 

`bundle install`
 
## Install JavaScript packages

Run the command:

`yarn`

## Set up database

Copy the file `/config/database.yml.example` and rename it to `/config/database.yml`, in this file place your PostgreSQL configuration. Later, run the command:

`rails db:setup`

This command will create the database, load the schema, and initialize it with the seed data.

If there are pending migrations, also run the command:

`rails db:migrate`

## Run the Rails server

Run the command:

`rails s`

or

`rails server`

When the server is running, try to access [http://localhost:3000](http://localhost:3000)
  
## Screenshots 

### Mobile
![imagen](https://user-images.githubusercontent.com/12243331/137821323-24db7b97-1e9b-4a05-b3e7-71c68ede1b81.png)

### Tablet
![imagen](https://user-images.githubusercontent.com/12243331/137821367-6c1bb33b-faad-4f97-9e8b-d26586444413.png)

![imagen](https://user-images.githubusercontent.com/12243331/137821546-5b319d9c-48b8-48c8-a74a-470937aa7656.png)

### Desktop
![imagen](https://user-images.githubusercontent.com/12243331/137821436-b9f0bc3c-e8e7-4475-8c39-54cdc9492cf2.png)

YilKanda committed:
Video testimonial section

A section was made where you can see a testimony video of one of the first graduates of the BrightCoders program.

xFerchoVJ reviewed (dismissed):
LGTM!

OswaldoPineda reviewed (commented):
I recommend using the 7-1 architecture for your styles

jairolan reviewed (dismissed):
LGTM

YilKanda committed:
Replacement of the video call locally, by the video call externally.

I replace the way to call the video, using calling the video from Google Drive instead of calling it from the Assets folder.
